pandas/_libs/tslibs/offsets.pyx

```diff
@@ -111,6 +111,14 @@ def as_datetime(obj: datetime) -> datetime:
     return obj
 
 
+cdef ABCTimestamp as_timestamp(obj):
```

Comment:
I thought we removed this previously? what is the purpose of this now? to centrailize the import?

Reply:
> to centrailize the import?

Yes.

Reply:
this would only have 2 times where you actually need to import Timestamp (and you don't even need to check if its ABC); i would just do that.

Reply:
updated+green